test/ServiceBus.Tests/EvictionStrategyTests/EHPurgeLogicTests.cs

```diff
@@ -60,146 +56,6 @@ public EHPurgeLogicTests()
             this.logger = new NoOpTestLogger().GetLogger(this.GetType().Name);
         }
 
-        //Disable tests if in netstandard, because Eventhub framework doesn't provide proper hooks for tests to generate proper EventData in netstandard
-#if BUILD_FLAVOR_LEGACY
```

Comment:
@xiazen @jason-bragg can you validate if removing these is the right call now that we are fully migrating to the new package? Maybe they provide the hooks now?

Reply:
please don't delete those tests. if they doesn't pass because netstandard, you can disable or comment them for now. I will come back and fix them.